### PR TITLE
Deploy settings

### PR DESCRIPTION
**Pull request by j-miyake**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,7 +24,6 @@ tmp/*
 /coverage/
 nbproject
 public/cache/*
-Gemfile.lock
```
Comment (yeuem1vannam) [inline .gitignore:27]: :+1:

File: config/unicorn.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+app_name = "publify"
+
+app_directory = "/usr/local/rails_apps/#{app_name}/current"
+working_directory app_directory # available in 0.94.0+
+
+listen "#{app_directory}/tmp/sockets/unicorn.sock", backlog: 128
+
+timeout 3000
+
+pid "#{app_directory}/tmp/pids/unicorn.pid"
```
Comment (yeuem1vannam) [inline config/unicorn.rb:12]: Is `"#{app_directory}/tmp/pids/` symlink to `shared_path` in `capistrano recipe`?

Comment (j-miyake) [inline config/unicorn.rb:12]: Yes, `config/deploy.rb:34` makes a symlink to shared dir.

``` ruby
set :linked_dirs, %w{bin log tmp/pids tmp/cache tmp/sockets vendor/bundle public/system}
```

Comment (yeuem1vannam) [inline config/unicorn.rb:12]: I got it :wink:

**Comment by yeuem1vannam** [comment]
OK

**Comment by linhnt** [comment]
ok

**Pull request merged**